area_a1_x: -0.092
area_a1_y: 1.850
area_a1_orientation: -0.066

area_a2_x: -0.354
area_a2_y: 1.579
area_a2_orientation: -3.084

area_a3_x: -0.129
area_a3_y: 1.164
area_a3_orientation: -0.055

area_a4_x: -0.432
area_a4_y: 0.894
area_a4_orientation: -3.109

area_b1_x: -0.109
area_b1_y: -0.555
area_b1_orientation: 0.045

area_b2_x: -0.234
area_b2_y: -0.308
area_b2_orientation: -3.106

area_b3_x: -0.012
area_b3_y: -1.263
area_b3_orientation: -0.159

area_b4_x: -0.218
area_b4_y: -1.212
area_b4_orientation: -3.135

casa_x: 0.049
casa_y: 0.070
casa_orientation: -0.003

buffer_x: -0.343
buffer_y: 0.258
buffer_orientation: 3.086
